script/validators/source_validator.py

```diff
@@ -66,6 +66,8 @@
     "src/codegen/util/cc_hash_table.cpp"
 ]
 
+TEST_CASE_PATT = re.compile(r'TEST_F\(([a-zA-Z]+), ([a-zA-Z]+)\)')
```

Comment:
Is this the right place to define the regex?

Reply:
I think so.